src/akamai/js/akamaiHD_flash.js

```diff
@@ -0,0 +1,1334 @@
+/*
+ * Akamai HD flash video plugin
+ */
+
+require("../../../html5-common/js/utils/InitModules/InitOO.js");
+require("../../../html5-common/js/utils/InitModules/InitOOUnderscore.js");
+require("../../../html5-common/js/utils/InitModules/InitOOHazmat.js");
+require("../../../html5-common/js/utils/constants.js");
+(function(_, $) {
+  var pluginName = "ooyalaFlashVideoTech";
+  var flashMinimumVersion = "11.1.0";
+  var cssFromContainer;
+
+  /**
+   * Config variables for paths to flash resources.
+   */
+
+  var pluginPath;
+  var filename = "akamaiHD_flash.*\.js";
+  var scripts = document.getElementsByTagName('script');
+  for (var index in scripts) {
+    var match = scripts[index].src.match(filename);
+    if (match && match.length > 0) {
+      pluginPath = match.input.match(/.*\//)[0];
+      break;
+    }
+  }
```

Comment:
i think this won't work for customers who concatinate all plugins into a single file or those who rename their files